src/api/rpc/index.ts

```diff
@@ -85,9 +85,12 @@ export async function getVkHash(
   vk: unknown,
 ): Promise<string> {
   try {
-    if (typeof vk !== 'object' || vk === null) {
+    if (
+      typeof vk !== 'string' &&
```

Comment:
Do we have a test case for a getVkHash by string?

Reply:
Just added on to the unit tests 👍